benchmarks/src/main/java/zipkin2/reporter/internal/AsyncReporterBenchmarks.java

```diff
@@ -77,10 +83,17 @@ public void clean() {
 
   @Setup(Level.Trial)
   public void setup() {
+    final BytesEncoder<Span> encoder = Stream
```

Comment:
Fixing the benchmark here, it was failing because encoding != SpanBytesEncoder.JSON_V2 (for PROTO3 & THRIFT)